lib/kitchen/driver/dokken.rb

```diff
@@ -648,11 +651,11 @@ def parse_registry_host(val)
       def pull_image(image)
         path = registry_image_path(image)
         with_retries do
-          if Docker::Image.exist?(path, {}, docker_connection)
-            original_image = Docker::Image.get(path, {}, docker_connection)
+          if Docker::Image.exist?(path, { "platform" => config[:platform] }, docker_connection)
+            original_image = Docker::Image.get(path, { "platform" => config[:platform] }, docker_connection)
           end
 
-          new_image = Docker::Image.create({ "fromImage" => path }, docker_creds_for_image(image), docker_connection)
+          new_image = Docker::Image.create({ "fromImage" => path, "platform" => config[:platform] }, docker_creds, docker_connection)
```

Comment:
@nrocco,

Just curious to know if there is any particular reason to change `docker_creds_for_image` to `docker_creds` here. This might cause the functionality added in #262 to not work anymore.

Reply:
Sorry that I haven't noticed it the first time.